server.js

```diff
@@ -4,8 +4,11 @@ const path = require('path');
 const app = express();
 const port = 3000;
```

Comment:
Здесь можно порт указать таким образом:
```suggestion
const PORT = process.env.PORT || 3000; 
```
Тогда будет возможность брать значение из env-переменных, а уже если там её не окажется, то брать значение по умолчанию